### PR TITLE
Style search error states

### PR DESCRIPTION
**Pull request by jvaclavik**
screenshot 2019-01-16 at 11 51 40

Btw you can notice that `Text` from `universal-components` with prop `weight="bold"` **is not bold** on iOS

### CODE REVIEW

File: packages/components/src/errorSearchIllustration/ErrorSearchIllustration.native.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+// @flow
+
+import React from 'react';
+import { Image } from 'react-native';
+import { StyleSheet } from '@kiwicom/universal-components';
+
+// $FlowFixMe - Cannot resolve module RelativeImageStub ( possible solution https://bit.ly/2BYfHRr )
+import ErrorSearchTravel from './assets/mobile/error-search-travel.png';
```
Comment (RobinCsl) [inline packages/components/src/errorSearchIllustration/ErrorSearchIllustration.native.js]: If you look here, it's also possible to use different pixel density pictures: https://github.com/kiwicom/mobile/tree/master/app/MMB/src/scenes/tripServices/insurance/paymentScene/images

Comment (JoseDuSV) [reply packages/components/src/errorSearchIllustration/ErrorSearchIllustration.native.js]: As I see there are already `@2x` and`@3x` versions inside assets subfolder, so I think it should be ok.

File: packages/components/src/ErrorInformation/index.js [hunk 2]
```diff
@@ -0,0 +1,3 @@
+// @flow
+
+export { default as ErrorInformation } from './ErrorInformation';
```
Comment (JoseDuSV) [inline packages/components/src/ErrorInformation/index.js]: This file is not necessary, after #98 discussion we settled on structure without extra `index.js` files.

File: packages/components/src/errorSearchIllustration/ErrorSearchIllustration.js [hunk 3]
```diff
@@ -0,0 +1,24 @@
+// @flow
+
+import React from 'react';
+
+const baseImagePath = './static/img/error-search-travel';
+
+export default function ErrorSearchIllustration() {
```
Comment (JoseDuSV) [inline packages/components/src/errorSearchIllustration/ErrorSearchIllustration.js]: It definitely shouldn't be part of this PR. But we can probably create issue after merge to combine this component with `searchIllustration` to some more universal `illustration` component. Since most code is shared between them.

Comment (jvaclavik) [reply packages/components/src/errorSearchIllustration/ErrorSearchIllustration.js]: I though about the same. We will create issue for it

File: packages/components/src/ErrorInformation/index.js (hunk 2 above)
Comment (jvaclavik) [reply packages/components/src/ErrorInformation/index.js]: OK, that's better :)

File: packages/components/src/errorSearchIllustration/ErrorSearchIllustration.js (hunk 3 above)
Comment (FilipMessa) [reply packages/components/src/errorSearchIllustration/ErrorSearchIllustration.js]: Where is the issue?

**Comment by RobinCsl** [comment]
Please @jvaclavik, look at #132 
We shouldn't be creating components in a different package with dependencies in a different one. This is going to break whenever we (re)move pictures from `apps/web/static` by mistake, thinking it's not used anywhere in that folder, and all the more so when there is another solution by using Kiwi.com's CDN.

**Comment by jvaclavik** [comment]
> Please @jvaclavik, look at #132
> We shouldn't be creating components in a different package with dependencies in a different one. This is going to break whenever we (re)move pictures from `apps/web/static` by mistake, thinking it's not used anywhere in that folder, and all the more so when there is another solution by using Kiwi.com's CDN.

OK, I understand, I didn't know that there is some CDN for it, I used the same approach as in search illustration and it worked :) 

But it would be much better to get it from CDN. How can I find URL of the image?

File: packages/components/src/errorSearchIllustration/ErrorSearchIllustration.js (hunk 3 above)
Comment (jvaclavik) [reply packages/components/src/errorSearchIllustration/ErrorSearchIllustration.js]: Probably nowhere when we will use images from CDN. It was about unifying illustration components.

**Comment by jvaclavik** [comment]
@JoseDuSV It should be fixed now

File: packages/relay/src/QueryRenderer.js [hunk 4]
```diff
@@ -22,9 +23,10 @@ export default class App extends React.Component<Props> {
   renderRelayContainer = ({ error, props }: ReadyState) => {
     if (error) {
       return (
-        <View>
-          <Text style={styles.text}>{error.message}</Text>
-        </View>
+        <IllustrationWithInformation
+          illustrationName="Error"
+          text={error.message}
```
Comment (JoseDuSV) [inline packages/relay/src/QueryRenderer.js]: In final version we will probably need some default error message instead of displaying exact error string (which can be quite confusing for user).
screenshot 2019-01-22 at 15 03 29

Or at least display some common "**Something went wrong**" header and then _error message_ underneath it with some small font and semi-transparent color.

But for now the current implementation is probably fine.

File: apps/core/scenes/results/EmptyResults.js [hunk 5]
```diff
@@ -1,13 +1,13 @@
...
+import { IllustrationWithInformation } from '@kiwicom/margarita-components';
 
 export default function EmptyResults() {
   return (
-    <View>
-      <Text>EmptyResults</Text>
-    </View>
+    <IllustrationWithInformation
+      illustrationName="NoResults"
+      text="Sorry, we couldn't find that connection."
```
Comment (JoseDuSV) [inline apps/core/scenes/results/EmptyResults.js]: Maybe we can also add `Try selecting different dates or nearby places.` suggestion like in web version. But it's totally up to you, I'm ok even with current state.

screenshot 2019-01-22 at 14 15 58

File: packages/relay/src/QueryRenderer.js (hunk 4 above)
Comment (jvaclavik) [reply packages/relay/src/QueryRenderer.js]: OK, done

File: apps/core/scenes/results/EmptyResults.js (hunk 5 above)
Comment (jvaclavik) [reply apps/core/scenes/results/EmptyResults.js]: Done (I changed the text according to new one in Figma)

Comment (jvaclavik) [reply apps/core/scenes/results/EmptyResults.js]: screenshot 2019-01-22 at 16 06 40

File: packages/relay/src/QueryRenderer.js (hunk 4 above)
Comment (jvaclavik) [reply packages/relay/src/QueryRenderer.js]: screenshot 2019-01-22 at 16 07 34
